cmd/minikube/cmd/start.go

```diff
@@ -1236,6 +1236,10 @@ func validateFlags(cmd *cobra.Command, drvName string) {
 		exit.Message(reason.Usage, "Sorry, please set the --output flag to one of the following valid options: [text,json]")
 	}
 
+	if viper.GetStringSlice(config.AddonListFlag) != nil && viper.GetBool(noKubernetes) {
```

Comment:
consider adding it to a helper func to fix the lint error "cyclomatic complexity 31"

helper could be caled
validateNoAddonsForNoKubernetes(config...)

Reply:
@medyagh 
- Is the function `validateNoAddonsForNoKubernetes()` correctly added ?
- I have called func `validateNoAddonsForNoKubernetes()` inside the func `provisionWithDriver()`, func `validateFlags()` was also called here, so is it a correct place to call it ?